plugin.video.retrospect/resources/libs/updater.py

```diff
@@ -0,0 +1,123 @@
+#===============================================================================
+# LICENSE Retrospect-Framework - CC BY-NC-ND
+#===============================================================================
+# This work is licenced under the Creative Commons
+# Attribution-Non-Commercial-No Derivative Works 3.0 Unported License. To view a
+# copy of this licence, visit http://creativecommons.org/licenses/by-nc-nd/3.0/
+# or send a letter to Creative Commons, 171 Second Street, Suite 300,
+# San Francisco, California 94105, USA.
+#===============================================================================
+
+import re
+
+from helpers.jsonhelper import JsonHelper
+from version import Version
+from logger import Logger
+
+
+class Updater(object):
```

Comment:
why is this needed if the plugin is in the official repository?

Reply:
I would like to notify users that enabled the beta/alpha notifications of new beta/alpha versions of the add-on. I don't have a private repository (due to bandwidth issues) so this just notifies those users that enabled these notifications in the settings.

Reply:
Fine by me.